README.md

```diff
@@ -1 +1,34 @@
-# First-Portfolio
\ No newline at end of file
+# Creating my first Portfolio Website
+
+# Skills
+
+1. HTML
+2. CSS
+3. Gitlab
+4. Github
+
+# Instructions
+
+To set up my portfolio;
+
+- I created a new repository and opened a new branch.
+- I set up linters to the branch to ensure no syntax or styling errors.
+- I selected a template from figma that I would use to build my portfolio.
+- I used one of the templates from figma. its named " figma-template" and it is uploaded to my repository.
+- I used HTML for markup and CSS for styling.
+
+I have built two sections
+
+- Nav
+- Main
+
+# Tools
+
+Vscode
+Git
+Github
+
+# Author
+
+Github: https://www.github.com/kwambiee
+LinkeddIn: https://www.linkedin.com/in/joy-kwamboka
```

Comment:
- [ ] Please add other sections like Contributing, Acknowledgments, etc. You can use this [template](https://github.com/microverseinc/readme-template) but customize it for your project.